From c9dcb9745edaebff9e08100ae8f6ef927e0889be Mon Sep 17 00:00:00 2001
From: Hao Wei <hao.wei@amlogic.com>
Date: Fri, 27 Aug 2021 13:27:02 +0800
Subject: [PATCH] rdk: add irdeto hal config file. [1/1]

PD#RSP-2187

Problem:
irdeto hal need a config file.

Solution:
add irdeto hal config file.

Verify:
s905c2

Change-Id: I0bacac651dfa70de124be9a6ff711dd8c095adc2
Signed-off-by: Hao Wei <hao.wei@amlogic.com>
---
 etc/cas/irdeto/conf/irdeto_hal.conf | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 etc/cas/irdeto/conf/irdeto_hal.conf

diff --git a/etc/cas/irdeto/conf/irdeto_hal.conf b/etc/cas/irdeto/conf/irdeto_hal.conf
new file mode 100644
index 0000000..312f5b4
--- /dev/null
+++ b/etc/cas/irdeto/conf/irdeto_hal.conf
@@ -0,0 +1,14 @@
+#frontend
+frontend.ts.source=0
+frontend.device.count=1
+frontend.type.satellite=0
+frontend.type.cable=0
+frontend.type.terrestrial=1
+
+#demux export device num
+demux.device.num=3
+#demux id used by emm/ecm, use auto without add the item.
+#demux.device.index.emm=2
+
+#cca private data version
+cca.private_data.version=2
-- 
2.29.2

